### DISCUSSION TITLE
Storefront[3.0+] not responding on port 3000

### DISCUSSION THREAD

**Discussion by ayub02** [saleor/saleor#8112] [Q&A]
I used the latest commit [Fix storefront not running on platform with 3.0+](https://github.com/mirumee/saleor-platform/commit/9549f2f13e7158f229d5459fd5ca93c803c71c41) for docker-compose.yml. Now storefront is not showing up on 3000 and it doesn't throw any error during compilation. As seen in the image, storefront is not using any port. Logs are shown below.
Saleor: 3.1.0-a.5
Saleor-dashboard: 3.1.0-a.4
Saleor-storefront: 3.0.0-b.2
![image](https://user-images.githubusercontent.com/40331925/133597681-6587aff5-616e-45cc-811f-0d23b5aabad2.png)
![image](https://user-images.githubusercontent.com/40331925/133599020-2e60a2a4-30c8-4044-8d8b-aa6b0be76b7e.png)

**Comment by karolkielecki** [comment]
Thanks for reporting! We'll take a look into this as soon as we can.

**Comment by Ferferite** [comment]
Any news on this? I am facing the same issue
